rules/integrations/aws/defense_evasion_rds_instance_restored.toml

```
@@ -0,0 +1,70 @@
+[metadata]
+creation_date = "2021/06/29"
+integration = ["aws"]
+maturity = "production"
+updated_date = "2024/06/20"
+
+[rule]
+author = ["Austin Songer", "Elastic"]
+description = """
+An adversary with a set of compromised credentials may attempt to make copies of running or deleted RDS databases in order to evade defense mechanisms or access data. This rule identifies successful attempts to restore a DB instance using the RDS `RestoreDBInstanceFromDBSnapshot` or `RestoreDBInstanceFromS3` API operations. 
+"""
+false_positives = [
+    """
+    Restoring DB instances may be done by a system or network administrator. Verify whether the user identity, user agent,
+    and/or hostname should be making changes in your environment. Instance restoration by unfamiliar users or hosts
+    should be investigated. If known behavior is causing false positives, it can be exempted from the rule.
+    """,
+]
+index = ["filebeat-*", "logs-aws.cloudtrail-*"]
+language = "eql"
+license = "Elastic License v2"
+name = "AWS RDS DB Instance Restored"
+references = [
+    "https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_RestoreDBInstanceFromDBSnapshot.html",
+    "https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_RestoreDBInstanceFromS3.html",
+    "https://github.com/RhinoSecurityLabs/pacu/blob/master/pacu/modules/rds__explore_snapshots/main.py",
+    "https://cloud.hacktricks.xyz/pentesting-cloud/aws-security/aws-post-exploitation/aws-rds-post-exploitation#rds-createdbsnapshot-rds-restoredbinstancefromdbsnapshot-rds-modifydbinstance",
+]
+risk_score = 47
+rule_id = "bf1073bf-ce26-4607-b405-ba1ed8e9e204"
```

```suggestion
rule_id = "416358f3-778f-42fe-915e-6e59c0b16482"
```
changing rule id since I deprecated the old rule